bundle/edu.gemini.obslog/src/main/resources/openLog.jsp

```diff
@@ -38,7 +38,7 @@
                 </a>
             </display:column>
 
-            <display:column title="Last Modified" property="lastmodified" sortable="true"/>
+            <display:column title="Last Modified (UTC)" property="lastmodified" sortable="true"/>
```

Comment:
I suppose the sorting here will also be screwed up, but also think it isn't very high priority.

Reply:
Good catch.